main.py

```diff
@@ -0,0 +1,128 @@
+import cv2
+from matplotlib import image
+import mediapipe as mp
+import numpy as np
+
+mp_drawing = mp.solutions.drawing_utils
+mp_drawing_styles = mp.solutions.drawing_styles
+mp_holistic = mp.solutions.holistic
+mp_pose = mp.solutions.pose
+cap = cv2.VideoCapture('D:\\Tenis_04_07\\Wideonly\\zverev.mp4')
+print((int(cap.get(4)), int(cap.get(3))))
```

Comment:
Why does the potentially future user need to print capture size? If u need something for your debugging don't forget to remove it before commiting